README.md

```diff
@@ -1 +1,5 @@
-# cesium-webstorm-plugin
\ No newline at end of file
+# Cesium WebStorm Plugin
```

Comment:
Also say that this is developed by the Cesium team and include a logo, for example, https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/README.md